pyls/python_ls.py

```diff
@@ -294,10 +294,15 @@ def m_workspace__did_change_configuration(self, settings=None):
         for doc_uri in self.workspace.documents:
             self.lint(doc_uri)
 
-    def m_workspace__did_change_watched_files(self, **_kwargs):
-        # Externally changed files may result in changed diagnostics
+    def m_workspace__did_change_watched_files(self, changes=None, **_kwargs):
+        changed_py_files = set(d['uri'] for d in changes if d['uri'].endswith(('.py', '.pyi')))
```

Comment:
Should be add `.cfg` and `.ini` as well?

Reply:
I'd expect that this is handled by didChangeConfig, but on the other hand the client doesn't know that we're also accepting config files.

I'll look into what config file changes should trigger a lint.